examples/step-69/step-69.cc

```diff
@@ -0,0 +1,355 @@
+#include "helper.h"
```

Comment:
It's not going to compile...

Reply:
The header file is included on the branch :-)